cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java

```diff
@@ -48,7 +48,8 @@
  */
 public class NGSIMySQLSink extends NGSISink {
     
-    private static final String DEFAULT_ROW_ATTR_PERSISTENCE = "row";
+    private static final String MYSQL_QUOTE_CHAR = "`";
+	private static final String DEFAULT_ROW_ATTR_PERSISTENCE = "row";
```

Comment:
I have just spotted a little missindent in this line...

Reply:
done